src/bundle/Resources/views/themes/admin/ui/tab.html.twig

```diff
@@ -0,0 +1,3 @@
+<div id="ez-tab-{{ id }}" class="tab-pane ez-tab-content__pane {% if active|default(false) %}active{% endif %}">
```

Comment:
For me, the name of this file is misleading, a tab is a `<li class="nav-item ez-tabs__tab...` and not a `tab-pane` so I would change filename to `tab_pane.html.twig`

Reply:
```suggestion
<div id="ez-tab-{{ id }}" class="tab-pane ez-tab-content__pane {% if active|default(false) %}active{% endif %} "role="tabpanel" aria-labelledby="ez-tab-label-{{ id }}">
```